website/docs/secrets/setup-secrets-operator.mdx

```diff
@@ -3,12 +3,180 @@ title: Setup Secrets Operator
+        --source=secrets \
+        --path=./secrets/gpg \ # Path for the secrets on the git repository
+        --prune=true \
+        --interval=10m \
+        --decryption-provider=sops \
+        --decryption-secret=sops-gpg-private-key    # Private key secret name on the cluster
+    ```
+
+- Annotate the kustomization object created in the previous step with the name and namespace of the public key.
+
+    ```yaml
+    kubectl annotate kustomization gpg-secrets \
+        sops-public-key/name=sops-gpg-public-key \ # Public key secret name on the cluster
+        sops-public-key/namespace=flux-system      # Public key secret namespace on the cluster
+    ```
+
+- The kustomization object should looks like this
+
+    ```yaml
+    apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
+    kind: Kustomization
+    metadata:
+    name: gpg-secrets
+    namespace: flux-system
+    annotations:
+        sops-public-key/name: sops-gpg-public-key
+        sops-public-key/namespace: flux-system
+    spec:
+    interval: 10m
+    sourceRef:
+        kind: GitRepository
+        name: secrets
+    path: ./secrets/gpg
+    decryption:
+        provider: sops
+        secretRef:
+        name: sops-gpg-private-key
+    prune: true
+    validation: server
+    ```
+
+import SopsBootstrapJob from "!!raw-loader!./assets/sops-bootstrap-job.yaml";
+import TemplateParams from "!!raw-loader!./assets/template-params.yaml";
+import TemplateAnnotations from "!!raw-loader!./assets/template-annotations.yaml";
+
+### Bootstrapping SOPS to leaf clusters
+
+Bootstrapping SOPS to leaf cluster in WGE by utilizing ClusterBootstrapConfig job to bootstrap Flux and SOPS.
+The job is a container which generates SOPS secrets key pairs, Creates secrets with the private key, Creates a secret with the public key (to be used in self-serve flow) and the proper Rbac for it.
+As well as an option to push the public key to the git repository via PR (to be distributed)
+
+### Prerequisites
+
+#### ClusterBootstrapConfig job
+
+The following example is using GPG encryption to install SOPS and generate keys when bootstrapping leaf clusters. Create the following ClusterBootstrapConfig CR and push it to your fleet repo.
+        
+<details><summary>Expand to view </summary>
+    <CodeBlock
+    title="clusters/management/capi/boostrap/sops-bootstrap-job.yaml"
+    className="language-yaml"
+    >
+        {SopsBootstrapJob}
+    </CodeBlock>
+</details>
+
+#### Cluster template updates
+
+In order to bootstrap SOPS to leaf clusters, we need some modifications to the cluster template to allow creating a [Kustomization](https://fluxcd.io/flux/guides/mozilla-sops/#configure-in-cluster-secrets-decryption)
+for reconciling the secrets on the cluster using SOPS and to run the `ClusterBootstrapConfig` job during cluster creation.
+
+- The template metadata should have annotation `templates.weave.works/sops-enabled: "true"`. This will be used by WGE to create the Kustomization with the cluster files.
```

Comment:
I would remove all the bullet points in this section, they dont add much to the explanation and are a bit confusing visually. You can just use the text as it is with code blocks under each sentence like you did.